content/realtime/channel-metadata.textile

```diff
@@ -19,19 +19,19 @@ h2(#overview). Overview
 
 Every channel in Ably's Data Stream Network has "lifecycle events":#lifecycle-events. The "channel lifecycle events":#lifecycle-events are triggered whenever a channel changes its "lifecycle state":#lifecycle-events. The "occupancy":#occupancy events are triggered whenever there is a change in the number occupants in a channel, indicating how many of each type of occupant there are. Both the "channel lifecycle":#lifecycle-events and the "occupancy":#occupancy events are published onto special channels called "metachannels":#metachannels, that can be subscribed to with our "realtime libraries":/realtime. This is often used in conjunction with the "Ably Reactor":/reactor, which allows you to send this data to your "serverless endpoints":/general/functions, "queues":general/queues, and "streams":general/firehose.
```

Comment:
The complete overview will need to be updated, unfortunately so the readers understand what the API is for, the nomenclature and the levels of this metadata as per what we discussed.